cmd/golembase/account/create/create.go

```diff
@@ -1,57 +1,99 @@
 package create
 
 import (
+	"bufio"
 	"errors"
 	"fmt"
+	"io"
 	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
 
 	"github.com/adrg/xdg"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
 	"github.com/ethereum/go-ethereum/cmd/golembase/account/pkg/useraccount"
-	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/urfave/cli/v2"
+	"golang.org/x/term"
 )
 
 func Create() *cli.Command {
 	return &cli.Command{
 		Name:  "create",
 		Usage: "Create a new account",
 		Action: func(c *cli.Context) error {
-			privageKeyPath, err := xdg.ConfigFile(useraccount.PrivateKeyPath)
+			// This creates the directories if they don't already exist
+			walletPath, err := xdg.ConfigFile(useraccount.WalletPath)
 			if err != nil {
-				return fmt.Errorf("failed to get config file path: %w", err)
+				return fmt.Errorf("failed to create config file path: %w", err)
 			}
 
-			fmt.Println("privageKeyPath", privageKeyPath)
+			fmt.Println("walletPath", walletPath)
 
-			privageKeyBytes, err := os.ReadFile(privageKeyPath)
-			switch {
-			case errors.Is(err, os.ErrNotExist):
-				privateKey, err := crypto.GenerateKey()
-				if err != nil {
-					return fmt.Errorf("failed to generate private key: %w", err)
-				}
-				privateKeyBytes := crypto.FromECDSA(privateKey)
-				err = os.WriteFile(privageKeyPath, privateKeyBytes, 0600)
-				if err != nil {
-					return fmt.Errorf("failed to write private key: %w", err)
-				}
+			if info, err := os.Stat(walletPath); err == nil && info.Size() != 0 {
+				return fmt.Errorf("A wallet already exists at %s", walletPath)
+			}
```

Comment:
the branch when there is an error returned from `os.Stat` is not handled properly